packages/backend/src/helpers/setupPassportStrategies.js

```diff
@@ -0,0 +1,21 @@
+const RedditStrategy = require('passport-reddit').Strategy;
+
+const setupPassportStrategies = passport => {
+  passport.use(
+    new RedditStrategy(
+      {
+        clientID: 'rWjd7fOjL7eq3w',
```

Comment:
They are bound to my reddit account and the callback has to be specified in Reddit, but Reddit allows modifying the callback URL.